package.json

```diff
@@ -37,6 +37,7 @@
     "@types/feedparser": "^2.2.5",
     "@types/jest": "^29.5.0",
     "@types/react": "^18.0.28",
+    "@types/react-test-renderer": "^18.0.0",
```

Comment:
We don't directly use `react-test-renderer`, but it is indirectly used by `@testing-library/react-native`. Namely the `ReactTestInstance` type is used in unit tests, but Typescript doesn't provide any info about this type (and assumes it to be like `any`), unless we have this `@types/` package added. If `@testing-library/react-native` ever changes to correctly export the `ReactTestInstance` type then we can remove this package.